public/js/mainCtrl.js

```diff
@@ -0,0 +1,41 @@
+alwaysDo.controller('MainCtrl', function() {
+
+  self = this;
```

Comment:
Great pattern here - reference `self` to `this` so you can always access the top-level scope. NICE